### PR TITLE
Add instance inspect event

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

**Comment by alena1108** [comment]
LGTM

File: cattle/plugins/docker/compute.py [hunk 1]
```diff
@@ -716,3 +716,24 @@ def _do_instance_remove(self, instance, host, progress):
...
+        try:
+            container_id = instanceInspectRequest.id
+            container = self.get_container_by(client, lambda x:
+                                              self._id_filter(container_id, x))
+        except (KeyError, AttributeError):
+            pass
+
+        try:
+            name = '/{0}'.format(instanceInspectRequest.name)
+            container = self.get_container_by(client, lambda x:
```
Comment (sonchang) [inline cattle/plugins/docker/compute.py]: If the container was already found by the id, do we want to intentionally override it?

**Comment by sonchang** [comment]
Besides my one question, LGTM

File: cattle/plugins/docker/compute.py (hunk 1 above)
Comment (cjellick) [inline cattle/plugins/docker/compute.py]: No. will fix. Good catch.

**Pull request merged**